Tests/src/PublicAPITests/SpaceSystemTests.cpp

```diff
@@ -1346,7 +1346,7 @@ CSP_PUBLIC_TEST(CSPEngine, SpaceSystemTests, SetUserRoleOnInviteTest)
 	CreateSpace(SpaceSystem, UniqueSpaceName, TestSpaceDescription, SpaceAttributes::Private, nullptr, nullptr, nullptr, Space);
 
 	// Invite second test account as a Moderator Role user
-	auto [Result] = AWAIT_PRE(SpaceSystem, InviteToSpace, RequestPredicate, Space.Id, AlternativeLoginEmail, true, "https://www.magnopus.com/");
+	auto [Result] = AWAIT_PRE(SpaceSystem, InviteToSpace, RequestPredicate, Space.Id, AlternativeLoginEmail, true, "");
```

Comment:
Why did this line need changing?

Reply:
Hm, wouldn't that mean other PR's checks would be failing similarly?

Reply:
Yeah I'd love to know why - any ideas @jeromagnopus?

Reply:
Marking this as resolved to keep things moving, but I would really like to understand this!

Reply:
I'm not sure what this parameter is meant to be - but I'm assuming it's the `emailLinkUrl` that we use to construct the environment and release-build-specific URL?  If so, we have an allow-list for these and `https://www.magnopus.com` is not in them.  This is somewhat recent, but was merged and deployed weeks ago at minimum.  

I'm not sure why you're suddenly seeing failures, however, a solution would be to change to `https:\\dev.magnoverse.space`

Reply:
@MAG-SamBirley Please see what I wrote above - not sure if it answers all questions, however.